torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1378,6 +1378,21 @@ def _make_tensor_helper(shape, low=None, high=None):
 
     return samples
 
+def sample_inputs_entr(op_info, device, dtype, requires_grad):
+    low, high = op_info.domain
```

Comment:
Because this knows it's providing sample inputs for entr() it can set low and high directly